src/app/common/utils/helpers/stringify.js

```diff
@@ -0,0 +1,5 @@
+import { isString } from 'app/common/utils/helpers/guards';
+
+export default function stringify(input) {
+  return isString(input) ? input : JSON.stringify(input, null, 2);
```

Comment:
From multiple examples of the duplicated code I copied the one which hand't had `trim` call. Lucky me (not lucky) =(

Thanks for the report, I think I'll return that trim and publish another bugfix version later today. Just in case...